Docs/User Guide/User Guide.md

```diff
@@ -10,6 +10,10 @@ A valid hardware configuration for the core must be provided to each instance of
 
 ![BTI XML Editor](Screenshots/BTI_XML_Configuration.PNG)
 
+Additionally, the `name` attribute for each label in the hardware configuration file should adhere to the following format in order to be recognized by the custom device: name="label #" where # is the `labelDecimal`, e.g. "label 16".  If the label uses SDI=10, the label should also include `_2` at the end of the label, e.g. "label 16_2".  The `BTI XML Editor` will generate these label names by default.
```

Comment:
The section on SDI should be more generic, not just for SDI = 2. The added digit should be the decimal value of the configured SDI. So `_0` for 00, `_1` for 01, `_3` for 11.

Reply:
updated in commit 2